docs/fundamentals/dashboard/standalone-for-python.md

```diff
@@ -0,0 +1,237 @@
+
+> [!div class="checklist"]
+>
+> - Start the Aspire dashboard in standalone mode.
+> - Use the Aspire dashboard with a Python application.
+
+## Prerequisites
+
+To complete this tutorial, you need the following:
+
+- [Docker](https://docs.docker.com/get-docker/)
+  - You can use an alternative container runtime, but the commands in this article are for Docker.
+- [Python 3.9 or higher](https://www.python.org/downloads/) locally installed
+- A sample application
+
+## Sample application
+
+This quickstart can be completed using either Flask, Django, or FastAPI. A sample application in each framework is provided to help you follow along with this quickstart. Download or clone the sample application to your local workstation.
+
+### [Flask](#tab/flask)
+
+```Console
+git clone https://github.com/Azure-Samples/msdocs-python-flask-webapp-quickstart
+```
+
+### [FastAPI](#tab/fastapi)
+
+```Console
+git clone https://github.com/Azure-Samples/msdocs-python-fastapi-webapp-quickstart.git
+```
+
+---
+
+To run the application locally:
+
+### [Flask](#tab/flask)
+
+1. Go to the application folder:
+
+    ```Console
+    cd msdocs-python-flask-webapp-quickstart
+    ```
+
+1. Create a virtual environment for the app:
+
+    [!INCLUDE [Virtual environment setup](./includes/standalone/virtual-environment-setup.md)]
+
+1. Install the dependencies:
+
+    ```Console
+    pip install -r requirements.txt
+    ```
+
+1. Run the app:
+
+    ```Console
+    flask run
+    ```
+
+1. Browse to the sample application at `http://localhost:5000` in a web browser.
+
+    :::image type="content" source="./media/standalone/run-flask-app-localhost.png" alt-text="Screenshot of the Flask app running locally in a browser":::
+
+### [FastAPI](#tab/fastapi)
+
+1. Go to the application folder:
+
+    ```Console
+    cd msdocs-python-fastapi-webapp-quickstart
+    ```
+
+1. Create a virtual environment for the app:
+
+    [!INCLUDE [Virtual environment setup](./includes/standalone/virtual-environment-setup.md)]
+
+1. Install the dependencies:
+
+    ```Console
+    pip install -r requirements.txt
+    ```
+
+1. Run the app:
+
+    ```Console
+    uvicorn main:app --reload
+    ```
+
+1. Browse to the sample application at `http://localhost:8000` in a web browser.
+
+    :::image type="content" source="./media/standalone/run-fastapi-app-localhost.png" alt-text="Screenshot of the FastAPI app running locally in a browser.":::
+
+---
+
+## Adding OpenTelemetry
+
+To use the Aspire dashboard with your Python application, you need to install the OpenTelemetry SDK and exporter. The OpenTelemetry SDK provides the API for instrumenting your application, and the exporter sends telemetry data to the Aspire dashboard.
+
+1. Install the OpenTelemetry SDK and exporter:
+
+    ```Console
```

Comment:
```suggestion
    ```python
```

Reply:
`pip` is a CLI tool (like NuGet), which is why it's `Console` (or should be `console`).

```suggestion
    ```console
```